docs/walkthroughs/local-setup.adoc

```diff
@@ -0,0 +1,179 @@
+[[local-setup]]
+Local Setup
+-----------
+
+This document will walk you through setting up a locally running Mobile
+Control Panel and a local OpenShift cluster using `oc cluster up`. This
+guide can be followed for _both_ local development and just a local
+installation.
+
+[[requirements]]
+Requirements
+~~~~~~~~~~~~
+
+* http://docs.ansible.com/ansible/latest/intro_installation.html[Ansible]
+>= 2.3.2.0
+* https://hub.docker.com/[Docker Hub account]
+* https://docs.docker.com/engine/installation/[Docker]
+* https://github.com/openshift/origin/releases[oc command line client]
+* https://nodejs.org/en/[Node.js] >= 6.10.0
+
+*Note:* On Linux you also need to have the `libselinux-python` package
+installed!
+
+For local development, you will need to have a Go environment setup:
+
+[source,sh]
+----
+export PATH="$PATH:~/go/bin"
+----
+
+You will want to add the path permanently to your `.bashrc` or
+`.bashprofile`.
+
+[[creating-a-local-cluster]]
+Creating a Local Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+[[clone-this-repository]]
+Clone this repository
+^^^^^^^^^^^^^^^^^^^^^
+
+[source,bash]
+----
+git clone git@github.com:feedhenry/mcp-standalone.git
+----
+
+*Note:* it is a good idea to clone this into a valid $GOPATH, however it
+is not essential.
+
+[[install-the-required-ansible-dependencies]]
+Install the required ansible dependencies:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+[source,sh]
+----
+ansible-galaxy install -r ./installer/requirements.yml
+----
+
+[[firewall-setup]]
+Firewall setup
+^^^^^^^^^^^^^^
+
+Next we need to configure Docker registry _and_ ports required as part
```

Comment:
I'm guessing this is meant to be bullet points? For me locally and on GitHub it is rendering as a single line including the *'s . A newline after "cluster setup:" seemed to make it render correctly.

Reply:
:+1: